source/rust_verify/example/playground.rs

```diff
@@ -1,37 +1,25 @@
-use builtin_macros::*;
```

Comment:
Please remove this change.

Reply:
Sorry I should really avoid using `git add .`